src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-form/user-form.component.html

```diff
@@ -103,13 +103,26 @@
                   i18n>Password confirmation doesn't match the password.</span>
           </div>
         </div>
-
         <!-- Password expiration date -->
         <div class="form-group row"
-             *ngIf="showExpirationDateField()">
+             *ngIf="!authStorageService.isSSO()">
           <label class="col-form-label col-sm-3"
                  for="pwdExpirationDate">
             <ng-container i18n>Password expiration date</ng-container>
+            <cd-helper class="text-pre"
+                       *ngIf="pwdExpirationSettings.pwdExpirationSpan == 0">
+              <p>
+                The Dashboard setting defining the expiration interval of
+                passwords is set to <strong>0</strong> currently. It means the
+                user password will only expire once.
```

Comment:
```
The Dashboard setting defining the expiration interval of
passwords is currently set to <strong>0</strong>. This means that, 
if a date is set, the user password will only expire once.
```

Reply:
Thanks! Would you also be fine with:

```
The Dashboard setting defining the expiration interval of
passwords is currently set to <strong>0</strong>. This means
if a date is set, the user password will only expire once.
```

?

Reply:
Sure.